docs/book/src/getting_started/README.md

```diff
@@ -95,3 +111,10 @@ Now run `trunk serve --open` from the root of the `leptos-tutorial` directory.
 Trunk should automatically compile your app and open it in your default browser.
 If you make edits to `main.rs`, Trunk will recompile your source code and
 live-reload the page.
+
+
+Welcome to the world of UI development with Rust and WebAssembly (WASM), powered by Leptos and Trunk!
+
+---
+
+Now before we get started building your first real UI's with Leptos, there's a couple of things you might want to know to help make your experience with Leptos just a little bit easier.
```

Comment:
`there's a couple of things` -> `there are a couple of things`
or `there're a couple of things` 🤔 lol

Reply:
No problem. And no judgement here!